### PR TITLE
[quazip/x.x.x] Use version ranges for dependencies

### PR DESCRIPTION
**Pull request by MartinDelille**
Specify library name and version:  **lib/1.0**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/quazip/all/conanfile.py [hunk 1]
```diff
@@ -48,10 +48,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("qt/5.15.9")
-        self.requires("zlib/1.2.13")
+        self.requires("qt/[>=5.15.2]")
```
Comment (CJCombrink) [inline recipes/quazip/all/conanfile.py:51]: Does this mean that this package is Qt 6 compatible?

**Review by CJCombrink — APPROVED** [review]

File: recipes/quazip/all/conanfile.py (hunk 1 above)
Comment (MartinDelille) [reply recipes/quazip/all/conanfile.py:51]: Thank you for the review! Yes it is compatible. I'm just wondering which qt version will be picked up in the test.

That fact that the target name contains the qt version can be problematic though.

**Comment by RubenRBS** [comment]
Hi @MartinDelille thanks a lot for your contribution. I see that you closed the PR, but let me give you some insight:

Our currently policy for version ranges is to limit them to a handful of libraries for which we know and understand their compatibility matrixes. Once we're comfortable having more libraries use this method, we'll let everyone know so it can be applied elsewhere :)

**Comment by MartinDelille** [comment]
I closed this PR because I had an issue with conan v2 and quazip/qt version conflict. I fixed it by using the requires override attribute!

**Pull request closed without merging**